src/java/org/apache/cassandra/service/accord/serializers/CommandSerializers.java

```diff
@@ -99,6 +100,13 @@ public void serialize(T ts, DataOutputPlus out, int version) throws IOException
             TopologySerializers.nodeId.serialize(ts.node, out, version);
         }
 
+        public void serialize(T ts, DataOutputPlus out) throws IOException
```

Comment:
why include ones without a version?  what caused this change?

Reply:
because we have some other callers that don't currently have a version, so easier to invoke ones without a version